src/rudderstack.ts

```diff
@@ -88,7 +97,10 @@ export class RudderStack {
   track<T extends keyof TEvents>(event: T, payload: TEvents[T]) {
     if (this.has_initialized && this.has_identified) {
       try {
-        RudderAnalytics.track(event, payload)
+        RudderAnalytics.track(event, {
+          account_type: this.account_type,
+          ...(payload || {}),
+        })
```

Comment:
accounty_type pass resolved in this function
`    const setCoreAnalyticsData = (data: TCoreTrackData) => {
        _rudderstack.identifyEvent(getId(), { language: data.language })
        // @ts-ignore
        delete data['language']
        coreData = { ...data }
    }`
why do we need this here as well? and how we can set this value?
I saw the similar way for app.deriv.com, but this package isolated, isn't it?

Reply:
we can set the value of `accout_type` from the above method `setAccountType` , this is how it's done in `deriv-app`

Reply:
I did it here, please check https://github.com/binary-com/deriv-analytics/pull/20